pkg/kubelet/cm/cpumanager/state/state_file_test.go

```diff
@@ -77,13 +77,15 @@ func TestFileStateTryRestore(t *testing.T) {
 		stateFileContent string
 		policyName       string
 		expErr           string
+		expPanic         bool
 		expectedState    *stateMemory
 	}{
 		{
-			"Invalid JSON - empty file",
+			"Invalid JSON - one byte file",
 			"\n",
 			"none",
-			"state file: could not unmarshal, corrupted state file",
+			"[cpumanager] state file: unable to restore state from disk (unexpected end of JSON input)",
+			true,
 			&stateMemory{
```

Comment:
nit: `expectedState` not needed at all when `expPanic` is true

Reply:
Good point, will update

Reply:
Updated